.github/workflows/rolling-release.yml

```diff
@@ -0,0 +1,96 @@
+name: Rolling Release Workflow
```

Comment:
Seems like there's quite a lot of duplicated code with the versioned release code here, wondering if there are ways to consolidate it.

Reply:
I completely agree! I was actually planning to delay this refactoring until I put those actions in the common Delphix action directory to be used by multiple projects (probably `sdb` and `drgn`). Do you think it would be preferable to try it now?

Reply:
It's up to you. If you want to add this for both `sdb` and `drgn` it would definitely be better to put this into delphix-actions.

Reply:
Yeah that's the plan I just wanted to have it in this repo first before refactoring it to be used by multiple repos as I'm still experimenting with things.

Reply:
Sure, you have my approval in any case.